ReadMe.md

```diff
@@ -64,7 +64,7 @@ $ mkdir build
 
 ```bash
 $ cd build
-$ cmake -DCMAKE_INSTALL_PREFIX=/opt/adios2/2.1.1/gnu/openmpi ../source
+$ cmake -DCMAKE_INSTALL_PREFIX=/opt/adios2/2.1.1/gnu/openmpi ../ADIOS2
```

Comment:
Still a mismatch.  This clones into the `source` directory but then configures to the ADIOS2 directory